casper/src/test/scala/coop/rchain/casper/api/BlocksResponseAPITest.scala

```diff
@@ -171,86 +255,14 @@ class BlocksResponseAPITest
         _         <- casperRef.set(casperEffect)
         cliqueOracleEffect = SafetyOracle
           .cliqueOracle[Task](Sync[Task], logEff, metricsEff, spanEff)
-        blocksResponse <- BlockAPI.getBlocks[Task](Some(Int.MaxValue))(
+        blocksResponse <- BlockAPI.getBlocks[Task](Some(2))(
                            Sync[Task],
                            casperRef,
                            logEff,
                            cliqueOracleEffect,
                            blockStore
                          )
-      } yield blocksResponse.right.get.length should be(8) // TODO: Switch to 4 when we implement block height correctly
-  }
-
-  it should "return until depth" in withStorage { implicit blockStore => implicit blockDagStorage =>
```

Comment:
This test is gone? Should be a separate commit as well.

Reply:
No test has been removed